dotcom-rendering/src/web/layouts/NewsletterSignupLayout.tsx

```diff
@@ -101,6 +115,77 @@ const leftColWrapperStyle = css`
 	}
 `;
 
+const previewButtonWrapperStyle = css`
+	padding: ${space[2]}px 0;
+	${from.desktop} {
+		display: none;
+	}
+`;
+
+const mainGraphicWrapperStyle = css`
+	border-radius: ${space[2]}px;
+	overflow: hidden;
+	margin: ${space[4]}px 0;
+
+	& > figcaption {
+		display: flex;
+
+		${until.desktop} {
+			display: none;
+		}
+
+		background-color: ${brandAlt[400]};
+		align-items: center;
+		padding: ${space[1]}px;
+		${textSans.medium({ fontWeight: 'bold', lineHeight: 'tight' })}
+
+		svg {
+			margin-right: ${space[1]}px;
+			flex-shrink: 0;
+		}
+	}
+`;
+
+interface NewsletterDetailProps {
```

Comment:
For no particular reason, there's a convention to use `type` over `interface` when defining props 🤷 

```suggestion
type NewsletterDetailProps = {
```